doc/common_tools_writing_lints.md

```diff
@@ -78,14 +78,15 @@ impl LateLintPass<'_> for MyStructLint {
 There are two ways to do this, depending if the target trait is part of lang items.
 
 ```rust
-use clippy_utils::{implements_trait, match_trait_method, paths};
+use clippy_utils::{implements_trait, is_trait_method};
+use rustc_span::symbol::sym;
 
 impl LateLintPass<'_> for MyStructLint {
     fn check_expr(&mut self, cx: &LateContext<'_>, expr: &Expr<'_>) {
         // 1. Using expression and Clippy's convenient method
```

Comment:
These labels should also be updated to reflect what they are actually using, IMO. What do you think about these:
```rs
// 1. Using diagnostic items with the expression
// 2. Using lang items with the expression type
// 3. Using the type path with the expression
```